### ISSUE TITLE
Update the css, outdated direction syntax

### ISSUE THREAD

**Issue by karolyi** [usablica/intro.js#800]
Hey guys,

I'm using this project with webpack and autoprefixer, and with the latest update, I get the following warning:

```
WARNING in ./node_modules/css-loader?{"sourceMap":true}!./node_modules/postcss-loader/lib?{"config":{"path":"/project/postcss.config.js"},"sourceMap":true}!./node_modules/sass-loader/lib/loader.js?{"sourceMap":true,"includePaths":["/project/node_modules"]}!./frontend/src/scss/webshop/base.scss
(Emitted value instead of an instance of Error) autoprefixer: /project/node_modules/intro.js/introjs.css:12:2: Gradient has outdated direction syntax. Replace `cover` to `farthest-corner`.
```
It's just a warning, not an error, however it would be nice to have it fixed.

Any chance of a `.scss` version of the original css?

**Comment by bozdoz** [comment]
Assuming this is resolved.  Reopen if not.

**Comment by karolyi** [comment]
@bozdoz No it's not resolved, also I can't reopen the issue.

```
WARNING in ./node_modules/css-loader??ref--9-1!./node_modules/postcss-loader/lib??ref--9-2!./node_modules/sass-loader/lib/loader.js??ref--9-3!./frontend/src/scss/webshop/base.scss
(Emitted value instead of an instance of Error) autoprefixer: /Users/laszlokarolyi/Work/stoertebeker/ticketshop/node_modules/intro.js/introjs.css:12:2: Gradient has outdated direction syntax. Replace `cover` to `farthest-corner`.
```

**Comment by karolyi** [comment]
@bozdoz It'll be properly resolved when there's a new version released I guess, until then everyone sees the old one.

**Comment by karolyi** [comment]
@bozdoz any chance of getting a release with this fix contained?

**Comment by bozdoz** [comment]
I'm not sure of all the expectations for releases: version naming, sources, etc.  Would have to wait on @afshinm for that.

**Comment by bozdoz** [comment]
Again assuming this is resolved.  Reopen if not.
